Bottom: 4983cc1bd937531a9348ce33591bd8fdbce49d9b
Top:    6aaab879c0ee52739565cbc85ae16bfaa8f4a927
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-25 12:51:54 +0000

Copy contour is_round information when copying polygons


---

diff --git a/src/polygon1.c b/src/polygon1.c
index fd804b1..ca4dd28 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3517,6 +3517,10 @@ poly_CopyContour (PLINE ** dst, PLINE * src)
   (*dst)->xmin = src->xmin, (*dst)->xmax = src->xmax;
   (*dst)->ymin = src->ymin, (*dst)->ymax = src->ymax;
   (*dst)->area = src->area;
+  (*dst)->is_round = src->is_round;
+  (*dst)->cx = src->cx;
+  (*dst)->cy = src->cy;
+  (*dst)->radius = src->radius;
 
   for (cur = NEXT_EDGE (&src->head); cur != &src->head; cur = NEXT_VERTEX (cur))
     {
